### PR TITLE
Documentation correction

### PR DESCRIPTION
**Pull request by muxa**
Correction to small issues discovered when reading documentation

### CODE REVIEW

File: src/Documentation/Grain-Versioning/Compatible-grains.md [hunk 1]
```diff
@@ -47,5 +47,5 @@ with V1:
 
   - If the current activation is a V2 and the requested version is V1, the current activation will
     be able to process the request normally
-  - If the current activation is a V2 and the requested version is V1, the current activation will also
-    be able to process the request normally
\ No newline at end of file
+  - If the current activation is a V1 and the requested version is V2, the current activation will also
```
Comment (sergeybykov) [inline src/Documentation/Grain-Versioning/Compatible-grains.md]: Hmm, I think the original version of this sentence was correct. @benjaminpetit, what do you think?

Comment (muxa) [reply src/Documentation/Grain-Versioning/Compatible-grains.md]: The original version (line 50) practically said the same thing as the first line (line 48).

Comment (sergeybykov) [reply src/Documentation/Grain-Versioning/Compatible-grains.md]: Oh, yes, you are right, of course. :-)

**Comment by sergeybykov** [comment]
Thank you, @muxa!

**Pull request merged**